### DISCUSSION TITLE
Tiempo de ejecución - modificar compilador en makefile

### DISCUSSION THREAD

**Discussion by pascualferrer** [IIC2133-PUC/2024-1#146] [Tarea 2] [answered]
Hola,

Tengo una duda respecto al tiempo de ejecución y si es que no hay problema con comentar en el makefile el compilador OPT=-g y descomentar el OPT=-O3. Con este último mis tiempos de ejecución bajan considerablemente y están por debajo de los 10 segundos. No así con el primero, donde mis tiempos de ejecución sobrepasan por mucho los 10 segundos.

**Comment by sachondo7** [comment, marked as answer]
Si, no hay problema con modificarlo. Para revisar el codigo se corre con tu makefile
